src/OrchardCore.Modules/OrchardCore.Media/Drivers/MediaFieldDriver.cs

```diff
@@ -87,6 +87,18 @@ public override IDisplayResult Edit(MediaField field, BuildFieldEditorContext co
                     }
                 }
 
+                var filenames = field.GetAttachedFileNames();
```

Comment:
Why use extension method here? Why not just add property to MediaField?

Reply:
I did that because it was how the anchor was done.